### ISSUE TITLE
Zero division error in classify.py

### ISSUE THREAD

**Issue by marcverhagen** [clamsproject/app-swt-detection#49]
### Bug Description

Noticed by @keighrim in https://github.com/clamsproject/app-swt-detection/pull/45.

The app returns a ZeroDivisionError on one of the files. Stacktrace:

```
  File "/usr/local/lib/python3.8/site-packages/clams/restify/__init__.py", line 146, in post
    return self.json_to_response(self.cla.annotate(raw_data, **self.annotate_param_caster.cast(raw_params)))
	
  File "/usr/local/lib/python3.8/site-packages/clams/app/__init__.py", line 116, in annotate
    annotated = self._annotate(mmif, **runtime_params)
	
  File "/app/app.py", line 64, in _annotate
    predictions = self.classifier.process_video(vd.location)
	
  File "/app/modeling/classify.py", line 81, in process_video
    dur = round(fc / fps, 3) * 1000
```

The lines of code that are the problem are:

https://github.com/clamsproject/app-swt-detection/blob/f1ac468545d0985541ceed403a5ae9b9b2b9a989/modeling/classify.py#L78-L81

### Reproduction steps

```bash
$ podman  run -d --rm --name clams-app-swt-detection -p 5501:5000 -v /mnt:/mnt clams-app-swt-detection:v2  
$ clams source  video:/mnt/llc/llc_data/clams/wgbh/NewsHour/cpb-aacip-507-7659c6sk9k.mp4  | curl -d@- localhost:5501
```

**Comment by marcverhagen** [comment]
I can replicate this, but it does not seem to be an issue with `classifier.py` since I get no problems when I run the following locally (after getting the file):

```bash
$ python -m classify.py --input cpb-aacip-507-7659c6sk9k.mp4
```

**Comment by marcverhagen** [comment]
I now think this is an issue with the path used above, I have no problems if I do

```bash
$ clams source video:/data/video/cpb-aacip-507-7659c6sk9k.mp4  | curl -d@- localhost:5000
```

(waited a few minutes for the error to show up, it didn't so I interupted it).

**Comment by marcverhagen** [comment]
This still requires some changes in that the app should have given a proper error rather than letting it break when VideoCapture returns 0.0 as the frame rate of a video that does not exist (at least I think that is what the core of the problem is).

**Comment by marcverhagen** [comment]
When a non-existing file is opened with `cv2.VideoCapture()` then no error is raised. OpenCV will write a message to the standard output and return an empty VideCapture object. One way to report the error would be to change

https://github.com/clamsproject/app-swt-detection/blob/f1ac468545d0985541ceed403a5ae9b9b2b9a989/modeling/classify.py#L78-L81

by adding this after line 78:

```python
if not vidcap.isOpened():
    raise Exception(f'OpenCV could not open "{mp4_file}"')
```

A possibly better alternative is to use the `capture()` method in `mmif.utils.video_document_helper`, but that would be a little bit more work because `capture()` expects an instance of mmif.Document and it also at first sight it does not look like it traps this particular error either.

**Comment by keighrim** [comment]
fixed via #50.
